README.md

```diff
@@ -2,6 +2,24 @@
 
 Working with SQL the convenient way!
 
+# Using sql-workbench
+
+Currently, only MySQL is supported.
+
+## Connecting to a MySQL instance
+
+Run `M-x swb-connection-mysql`. It will prompt for `host`, `port`, `user`, `password`, and the database inside the MySQL server to use.
+
+The workbench buffer will open.
+
+## Using the workbench buffer
+
+The workbench buffer is the sql-workbench's main interface to the database. Here are some things you can do with it.
+
+* Write SQL statements in the buffer, then with point inside the sql statement, run `swb-send-current-query` (`C-c C-c`) to run that query on the database and send the 
```

Comment:
Seems like you cut the sentence in half here, or maybe github is going crazy with the rendering? :O